Kconfig

```diff
@@ -75,6 +75,15 @@ config KAS_INCLUDE_RT
 	default "kas/opt/preempt-rt.yml"
 	depends on PREEMPT_RT
 
+
+config IMAGE_SWUPDATE
+	bool "SWUpdate support for root partition"
```

Comment:
Does this actually smoothly works with the LXDE image?

Reply:
I rather suspect this is an alternative image, thus an entry for the choice above.